src/main/java/com/mentorship/notificationservice/service/NotificationServiceImpl.java

```diff
@@ -0,0 +1,30 @@
+package com.mentorship.notificationservice.service;
+
+import com.mentorship.notificationservice.domain.Notification;
+import com.mentorship.notificationservice.event.model.OrderEvent;
+import com.mentorship.notificationservice.repository.NotificationRepository;
+import jakarta.transaction.Transactional;
+import java.time.LocalDateTime;
+import lombok.RequiredArgsConstructor;
+import org.springframework.amqp.rabbit.annotation.RabbitListener;
+import org.springframework.stereotype.Service;
+
+@Service
+@Transactional
+@RequiredArgsConstructor
+public class NotificationServiceImpl {
```

Comment:
Let's rename this somehow like Listener or Processor. IMO it would be more understandable

P.S than we can change @Service to @Component